server/base/src/main/java/org/apache/accumulo/server/ServerContext.java

```diff
@@ -107,6 +111,9 @@ private ServerContext(ServerInfo info) {
     this.info = info;
     zooReaderWriter = new ZooReaderWriter(info.getSiteConfiguration());
     serverDirs = info.getServerDirs();
+
+    propStore = new ZooPropStore.Builder(info.getInstanceID(), zooReaderWriter,
```

Comment:
Why not call `ZooPropStore.initialize()` ?

Reply:
`ZooPropStore.initialize()` contains the exact same code. Also - ZooPropStore.initialize() is not called from anywhere. I'm thinking it was intended to be called from here.

Reply:
Addressed in 64191f7b1d - but maybe a different name would be clearer